url-bar.html

```diff
@@ -0,0 +1,70 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-location/iron-location.html">
+
+<!--
+`url-bar` is a helper element that displays a simple read-only URL bar if
+and only if the page is in an iframe. In this way we can demo elements that
+deal with the URL in our iframe-based demo environments.
+
+If the page is not in an iframe, the url-bar element is not displayed.
+
+@element url-bar
+@demo demo/url-bar.html
+-->
+<dom-module id='url-bar'>
+  <template>
+    <style>
+      :host {
+        margin: 0px;
+        padding: 15px 35px;
+        border-bottom: 2px solid gray;
+        height: 1em;
+        overflow-x: hidden;
+        overflow-y: hidden;
+        min-width: 600px;
```

Comment:
This giant min width is bad news bears for mobile, I think. Did you try it out on a small screen?